Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css

```diff
@@ -1,319 +1,122 @@
-  border-right: 1px solid #AAAAAA;
-  border-bottom: 1px solid #AAAAAA;
-  border-left: none;
-  background-color: #DDDDDD;
-  background-image: url(../Image/sprite.svg#TabbedMenuTabBackground);
-  background-size: 1px 33px;
-  background-repeat: repeat-x;
-  background-position: center;
-  color: Black;
-  font-size: 120%;
-  font-weight: bold;
-  text-decoration: none;
+  min-height: calc(var(--remotion-themed-spacing) * 7);
+  padding: 0 calc(var(--remotion-themed-spacing) * 4);
+  padding-top: var(--remotion-themed-spacing);
 }
 
-div.tabbedMainMenu span.tabStripTab a:focus,
-div.tabbedMainMenu span.tabStripTab a:hover,
-div.tabbedMainMenu span.tabStripTab a:active
-{
-  background-color: #FEFFFF;
-  background-image: url(../Image/sprite.svg#TabbedMenuTabBackgroundHover);
-  background-size: 1px 33px;
-  background-repeat: repeat-x;
-  background-position: top;
-  color: Black;
-  outline: none;
-}
-
-div.tabbedMainMenu span.tabStripTabSelected a
-{
-  border-top: 1px solid #AAAAAA;
-  border-right: 1px solid #AAAAAA;
-  border-bottom: none;
-  border-left: none;
-  background: none;
-  background-color: White;
-  color: Black;
-  font-size: 120%;
-  font-weight: bold;
-  text-decoration: none;
-}
-
-div.tabbedMainMenu span.tabStripTabSelected a:after
-{
-  content: '';
-  display: block;
-  /* Use a separate border element to ensure sharp corners instead of transitioning at the point where the border colors meet. */
-  border-bottom: solid 1px White;
-}
-
-div.tabbedMainMenu span.tabStripTabSelected a:focus,
-div.tabbedMainMenu span.tabStripTabSelected a:hover,
-div.tabbedMainMenu span.tabStripTabSelected a:active
-{
-  color: Black;
-  background: none;
-  background-color: White;
-  outline: none;
-}
-
-div.tabbedMainMenu span.tabStripTab.disabled a,
-div.tabbedMainMenu span.tabStripTab.disabled a:hover,
-div.tabbedMainMenu span.tabStripTab.disabled a:active
-{
-  color: #AAAAAA;
-  background: none;
-  background-color: #efefef;
-  text-decoration: none;
-}
-
-div.tabbedMainMenu span.tabStripTab.disabled img
-{
-  opacity: 0.4;
-  filter: progid:DXImageTransform.Microsoft.BasicImage(opacity=0.40),
-          progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
-}
-
-/* Content area of the tab */
-div.tabbedMainMenu li span.anchorBody
-{
-  padding-top: 0.3em;
-  padding-bottom: 0.3em;
-  padding-left: 0.6em;
-  padding-right: 0.6em;
-}
-
-div.tabbedMainMenu div.clearingPane
-{
-  clear: both;
-}
-
-/* Sub Menu Tabstrip */
-div.tabbedSubMenu div.tabStripTabsPane
-{
-}
+/* Main Menu Tabstrip and Sub Menu Tabstrip*/
 
 div.tabbedSubMenu div.tabStripTabsPane.empty
 {
-  padding-top: 0.4em; /* padding from tabStripTabWrapper, tabStripTab and content */
-}
-
-div.tabbedSubMenu span.tabStripTabSeparator
-{
-  display: none; /* magin-left must be set to 0 in the Non-IE browsers section for tabStripTab and tabStripTabSelected. */
+  min-height: calc(var(--remotion-themed-spacing) * 7);
 }
 
+div.tabbedMainMenu span.tabStripTab,
```

Comment:
How should the behavior be when long tab titles are used? where would the text wrap?

Reply:
If there is no space for the text in the current line, the whole tab jumps to the next line. However, if there is not enough space to display the text and the tab is the only tab on the line, it just wraps the text on whitespaces.

![image](https://user-images.githubusercontent.com/48289242/132456990-d6da846b-baf7-4183-bb73-3e79efbd378e.png)
![image](https://user-images.githubusercontent.com/48289242/132457025-475f1ad2-c3bf-4d8b-b4d0-f4aedb9b6089.png)
![image](https://user-images.githubusercontent.com/48289242/132457680-cb8fe52c-d85c-47f4-b4de-93acabbbb58f.png)